tank_lab_to_nwb/convert_towers_task/towersnwbconverter.py

```diff
@@ -70,6 +70,7 @@ def get_metadata(self):
             #         }
             #     }
             # },
+            SpikeGLXRecording=None,
```

Comment:
Metadata here should not be separated by DataInterface. It should be combined across DataInterfaces into one unified metadata object

Reply:
I know that's the approach we're starting to lean towards in development, but it's going to take a number of changes on those data interfaces in `nwb-conversion-tools` to be compatible. What you see here is what is compatible with the current master branch of `nwb-conversion-tools`. I'll gladly start work on the new format when we get all that together, though.